### ISSUE TITLE
Precommit check wants a license header on PNG files

### ISSUE THREAD

**Issue by skyostil** [google/trace-viewer#867]
$ git cl land
...
License must match:
.*? Copyright (\(c\) )?(2015|2014|2013|2012|2011|2010|2009|2008|2007|2006) The Chromium Authors\. All rights reserved\.\n.*? Use of this source code is governed by a BSD-style
 license that can be\n.*? found in the LICENSE file\.(?: \*/)?\n
Found a bad license header in these files:
  images/trace-viewer-circle-blue.png
  images/trace-viewer-circle-green.png
  images/trace-viewer-circle-red.png
  images/trace-viewer-circle-yellow.png
  images/trace-viewer-square-blue.png
  images/trace-viewer-square-green.png
  images/trace-viewer-square-red.png
  images/trace-viewer-square-yellow.png

**Comment by aschulm** [comment]
I ran into this too when I was adding some trace data. I need some input from others though on what the fix should be. The issue is we treat every != html as a "non_html_sources". Can we just replace that with a specific set of source extensions that we care about (e.g., .js)? If so, what should that set be. 

I've included a list of all extensions in the trace-viewer repo.

chromium
css
erb
gitignore
gn
gyp
gz
html
in
js
json
lock
log
markdown
md
png
py
pyc
rake
rb
settings
skp
svg
tmpl
trace
txt
yml
zip
